python/GafferUI/LabelPlugValueWidget.py

```diff
@@ -205,14 +210,18 @@ def __labelDoubleClicked( self, label, event ) :
 
 	def __labelEditingFinished( self, nameWidget ) :
 
-		with Gaffer.UndoScope( self.getPlug().ancestor( Gaffer.ScriptNode ) ) :
+		assert( len( self.getPlugs() ) == 1 )
+
+		plug = next( iter( self.getPlugs() ) )
+
+		with Gaffer.UndoScope( plug.ancestor( Gaffer.ScriptNode ) ) :
 			# Do what the NameWidget would have done for us anyway, so we
 			# can group it with the metadata deregistration in the undo queue.
-			self.getPlug().setName( nameWidget.getText() )
+			plug.setName( nameWidget.getText() )
 			# Remove any metadata label which would mask the name - if a user
 			# has gone to the trouble of setting a sensible name, then it should
 			# take precedence.
-			Gaffer.Metadata.deregisterValue( self.getPlug(), "label" )
+			Gaffer.Metadata.deregisterValue( plug, "label" )
```

Comment:
I don't think any of this is necessary is it? The old code would work fine, and even throw a helpful MultiplePlugsError if there were multiple, so we don't even need the assertion.

Reply:
Reverted [here](https://github.com/GafferHQ/gaffer/pull/3980/commits/bfef18130f35960ab9096b0e92e7b6ee9ab854bc).